config/liota.conf

```diff
@@ -10,6 +10,9 @@ uuid_path = /etc/liota/conf/uuid.ini
 [IOTCC_PATH]
 iotcc_path = /etc/liota/conf/iotcc.json
 
+[CORE_CFG]
+collect_thread_pool_size = 30
+
 [PKG_CFG]
 pkg_path = /etc/liota/packages
 pkg_msg_pipe = /var/tmp/liota/package_messenger.fifo
```

Comment:
As you are making the changes in this file, remove the extra lines from liota.conf end.